modules/swagger-codegen/src/main/resources/JavaJaxRS/cxf-cdi/enumClass.mustache

```diff
@@ -1,16 +1,33 @@
-import javax.xml.bind.annotation.XmlEnum;
-import javax.xml.bind.annotation.XmlType;
+@XmlType(name="{{datatypeWithEnum}}")
```

Comment:
@nickcmaynard  With the old version enumClass.mustache there are compile errors, as the generated sample files were out of date. Pls simply try with the old version and then you see the compilation errors.